kernel/Makefile

```diff
@@ -79,12 +84,15 @@ sources:
 # Arguments:
 # $1: Full kernel version, e.g., 4.9.22
 # $2: Kernel "series", e.g., 4.9.x
-# $3: Build a debug kernel (used as suffix for image)
+# $3: Build a specific kernel like -rt: Preempt-RT (used as suffix for image)
 # This defines targets like:
 # build_4.9.x and  push_4.9.x and adds them as dependencies
 # to the global targets
-# Set $3 to "-dbg", to build debug kernels. This defines targets like
+# Set $3 to "-rt", to build Preempt-RT kernels. This defines targets like
+# build_4.14.x-rt and adds "-rt" to the hub image name.
+# Set $4 to "-dbg", to build debug kernels. This defines targets like
 # build_4.9.x-dbg and adds "-dbg" to the hub image name.
+# Set $3 to "-rt" and $4 to "-dbg" to build debug Preempt-RT kernel.
 define kernel
 
 ifeq ($(3),)
```

Comment:
I think this now also needs a check on `$(4)` as I get:
```
Makefile:219: warning: overriding commands for target `sources/linux-4.14.19.tar.xz'
Makefile:218: warning: ignoring old commands for target `sources/linux-4.14.19.tar.xz'
Makefile:223: warning: overriding commands for target `sources/linux-4.9.81.tar.xz'
Makefile:222: warning: ignoring old commands for target `sources/linux-4.9.81.tar.xz'
```

Reply:
@rn Just like this?
 ifeq ($(3),)
+ifeq ($(4),)
 sources/linux-$(1).tar.xz: Makefile | sources
        curl -fsSLo sources/linux-$(1).tar.xz https://www.kernel.org/pub/linux/kernel/v4.x/linux-$(1).tar.xz 
 KERNEL_VERSIONS+=$(1)
 endif
+endif

Reply:
@rn Resent new pull request out.

Reply:
yes, something like this. Thanks.

Reply:
`ifeq ($(3)$(4),)` might be more concise and avoid the nested if.

Reply:
@ijc Sure. It looks good.